### PR TITLE
Add organization export & organization closure export enum source types

### PR DESCRIPTION
**Pull request by nanobash**

### CODE REVIEW

File: openapi/components/schemas/Files/FileCreateFromInline.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ properties:
     enum:
       - upload
       - camera
+      - organization_export
+      - organization_closure_export
```
Comment (ashkarpetin) [inline openapi/components/schemas/Files/FileCreateFromInline.yaml:29]:
```suggestion
      - organization-export
      - organization-closure-export
```

File: openapi/components/schemas/Files/FileCreateFromUrl.yaml [hunk 2]
```diff
@@ -26,6 +26,8 @@ properties:
     enum:
       - upload
       - camera
+      - organization_export
+      - organization_closure_export
```
Comment (ashkarpetin) [inline openapi/components/schemas/Files/FileCreateFromUrl.yaml:30]:
```suggestion
      - organization-export
      - organization-closure-export
```

File: openapi/components/schemas/Files/File.yaml [hunk 3]
```diff
@@ -20,6 +20,8 @@ properties:
     enum:
       - upload
       - camera
+      - organization_export
+      - organization_closure_export
```
Comment (ashkarpetin) [inline openapi/components/schemas/Files/File.yaml:24]: I don't think those types should be writable

File: openapi/components/schemas/Files/FileCreateFromInline.yaml (hunk 1 above)
Comment (nanobash) [reply openapi/components/schemas/Files/FileCreateFromInline.yaml]: Removed, thanks

File: openapi/components/schemas/Files/FileCreateFromUrl.yaml (hunk 2 above)
Comment (nanobash) [reply openapi/components/schemas/Files/FileCreateFromUrl.yaml]: Removed, thanks

File: openapi/components/schemas/Files/File.yaml (hunk 3 above)
Comment (nanobash) [reply openapi/components/schemas/Files/File.yaml:24]: Could you please tell me, can we specify somehow that upload/camera is `writable/readonly` and `organization_export/organization_closure_export` only readonly?

Comment (nanobash) [reply openapi/components/schemas/Files/File.yaml:25]: Do you mean this way? https://github.com/Rebilly/api-definitions/pull/987/files#diff-7e059a39200640c430800e893c497a03d7cd4cb3e0b524f77d156446117a013cR20

Comment (ashkarpetin) [reply openapi/components/schemas/Files/File.yaml:25]: There is a different schema for request, so readOnly might redundant here.

Comment (nanobash) [reply openapi/components/schemas/Files/File.yaml:25]: Should not we add them only for `Retrieve a File Record` response?

File: openapi/components/schemas/Files/File.yaml [hunk 4]
```diff
@@ -17,9 +17,12 @@ properties:
     description: The File source type.
     type: string
     nullable: true
+    readOnly: true
     enum:
       - upload
       - camera
+      - organization_export
+      - organization_closure_export
```
Comment (ashkarpetin) [inline openapi/components/schemas/Files/File.yaml:25]:
```suggestion
      - organization-export
      - organization-closure-export
```

Comment (nanobash) [reply openapi/components/schemas/Files/File.yaml]: Thanks, updated. Will update types on backend side with `-` as well

File: openapi/components/schemas/Files/File.yaml (hunk 3 above)
Comment (nanobash) [reply openapi/components/schemas/Files/File.yaml]: Discussed in person

**Comment by nanobash** [comment]
@Rebilly/php please review

**Comment by nanobash** [comment]
@Rebilly/php please review

**Review by Boydbueno — APPROVED** [review]

**Review by ninjasimon — APPROVED** [review]

**Pull request merged**